lib/wanda_web/views/result_view.ex

```diff
@@ -2,14 +2,19 @@ defmodule WandaWeb.ResultView do
   use WandaWeb, :view
 
   alias Wanda.Results.ExecutionResult
+  alias WandaWeb.ResultView
 
   def render("index.json", %{results: results, total_count: total_count}) do
     %{
-      items: render_many(results, WandaWeb.ResultView, "result.json", as: :result),
+      items: render_many(results, ResultView, "result.json"),
       total_count: total_count
     }
   end
 
+  def render("show.json", %{result: result}) do
```

Comment:
This looks like a smell. We have the `show.json` which should be used to render 1 result, and after that we have the `result.json` which is used for the same thing with other name.
I think we are missing something there. We should only have the `show` one there and do the mapping (the Map.put) there.

Reply:
I think this is how it should be, see: https://hexdocs.pm/phoenix/1.4.2/views.html#rendering-json